server/src/controllers/adminUser.controller.ts

```diff
@@ -83,4 +83,17 @@ export class AdminUserController {
     await this.adminUserRepository.updateById(id, adminUser);
   }
 
+  @del('/admin_user/{id}', {
```

Comment:
I thought we have deprecated this controller and just coding in auth.js?

Reply:
I did not realize this was deprecated I see now the /api requests are handled by the loopback request handler and the /auth by the express router . Thanks for letting me know. Is the intent to eventually move all API calls from the .ts controllers? Also should the controller adminUser.controller.ts be deleted/commented deprecated to ensure no one else makes the same mistake/uses this controller?

Reply:
Yes, @ericlvogt actually it's my fault, I should have removed these stuff away to avoid misunderstanding, could you comment this part? Then I will merge this PR to master.

Reply:
No problem @dadiorchen I should've noticed during development. I've commented all API request handlers for this controller but left the admin controller, model, and repository for now.